docker/Makefile

```diff
@@ -2,16 +2,19 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 targets := alma-9 centos-8 centos-9
-prefix := vdsm-test
+prefix := vdsm
+image_name := test
 
-.PHONY: $(targets) push
+.PHONY: $(targets) test push
 
 all: $(targets)
 
+$(image_name): all
+
 $(targets):
-	podman build -t $(prefix):$@ -f Dockerfile.$@ .
+	podman build -t $(prefix)-$(image_name):$@ -f Dockerfile.$@ .
 
```

Comment:
So that it can be invoked as `make ${{ matrix.container }}` in the containers Action. Acts as an alias to `make all`, and allows the name of the container to remain as `${{ matrix.prefix }}-${{ matrix.container}}`.

I do not think the script is confusing, is just an extra target and some naming flavor, which is very similar to what's done in the network makefile.